packages/govuk-frontend-review/src/common/nunjucks/filters/highlight.mjs

```diff
@@ -3,10 +3,10 @@ import hljs from 'highlight.js'
 /**
  * Format code with syntax highlighting
  *
- * @param {string} code - Code in plain text
+ * @param {string | undefined} code - Code in plain text
  * @param {string} [language] - Code programming language
  * @returns {string} Code with syntax highlighting
  */
-export function highlight (code, language) {
+export function highlight (code = '', language) {
```

Comment:
Do JS arguments use their defaults when passing `undefined`? 🤯 That's a bit arcane and it may be clearer to pass `(code ?? '').trim()` to `hljs.highlight` rather than lean on people knowing that bit of the specs 😊

Reply:
Yeah both the param value `undefined` or optional attribute will default to `''`

It's a free pass for `undefined` that `null` doesn't get

Happy to swap it over!

Reply:
Yeah, feels definitely safer in the long run to swap rather than rely on intricate knowledge of the specs to understand what's going on. Cheers for confirming the behaviour 😊

Reply:
I've removed this edit and added a new `inspect` filter to format JavaScript objects before highlighting

```njk
{{- stats | inspect | highlight("js") | safe if stats else "No build stats available" -}}
```